### PR TITLE
docs fixes

### LINKED ISSUE #9589: [docs] Document torch.bincount
`torch.bincount` is not available in the docs page, but the doc string has been added in `_torch_docs.py`. This was mentioned earlier in https://github.com/pytorch/pytorch/issues/4570#issuecomment-402002818

### PR DESCRIPTION
**Pull request by ssnl**
fixes #9589 #9507 #9502 #9390 #4176

### CODE REVIEW

**Review by soumith — APPROVED** [review]

File: docs/source/torch.rst [hunk 1]
```diff
@@ -258,6 +258,8 @@ Other Operations
 .. autofunction:: einsum
 .. autofunction:: flip
 .. autofunction:: histc
+.. autofunction:: bincount
+.. autofunction:: meshgrid
```
Comment (vishwakftw) [inline docs/source/torch.rst]: These are not in alphabetical order, could you please add them in the right order? The rendered docs don't sort them, unfortunately. Thanks.

Comment (SsnL) [reply docs/source/torch.rst]: done :)

Comment (vishwakftw) [reply docs/source/torch.rst]: Thank you. :smile:

**Comment by vishwakftw** [comment]
Do you want to consider documenting `scatter_add_` (https://github.com/pytorch/pytorch/issues/4176) as a part of this PR?

**Comment by SsnL** [comment]
@vishwakftw Added! Let me know if the doc I wrote can be improved.

File: torch/_tensor_docs.py [hunk 2]
```diff
@@ -1785,6 +1785,49 @@ def callable(a, b) -> number
...
 """)
 
+add_docstr_all('scatter_add_',
+               r"""
+scatter_add_(dim, index, other) -> Tensor
+
+Adds all values from the tensor :attr:`other` into :attr:`self` at the indices
+specified in the :attr:`index` tensor in a similar fashion as
+:meth:`~torch.Tensor.scatter_`. For each value in :attr:`other`, it is added to
+an index is specified by its index in :attr:`other` for dimension != :attr:`dim`
```
Comment (vishwakftw) [inline torch/_tensor_docs.py]: nit: ... index is specified ... --> ... index which is specified ...
nit?: != -->:math:`\neq`

**Review by vishwakftw — COMMENTED** [review]
Looks good to me, just one nit.

**Comment by vishwakftw** [comment]
Whoops, sorry about the delay.

**Comment by SsnL** [comment]
@vishwakftw no worries... the fb bot was too fast.

**Pull request closed without merging**